src/Goblin.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "Enemy.hpp"
+
+class Goblin : public Enemy {
+private:
+	const static int health;
+	const static int damage;
+	const static float damageCooldown;
+	const static float speed;
+	const static raylib::Rectangle collider;
+	static raylib::Texture* texture;
```

Comment:
we shouldnt override these things like damage, speed, collider, texture, as Enemy already has them. We can just use this->getTexture(), this->setTexture, etc.

Reply:
Since Goblin inherits Enemy there'll be duplicated variables for each

Reply:
this is not override. It's for Goblin to have it's own properties so that I can avoid passing all the properties when new it

https://github.com/xxori/uoa-oop-project/blob/79a429605739ac153c181e14fe66f91de10128d1/src/Goblin.cpp#L14

https://github.com/xxori/uoa-oop-project/blob/79a429605739ac153c181e14fe66f91de10128d1/src/Game.cpp#L39

Reply:
ah alright easy. maybe better to hardcode those values in the constructor so we can change them over time as difficulty increases. all good for now though